tests/unit/scripting.tcl

```diff
@@ -566,6 +577,11 @@ start_server {tags {"scripting"}} {
         assert_equal bar [run_script_ro {return redis.call('get', KEYS[1]);} 1 foo]
     }
 
+    test {EVAL_RO - Successful case via server.call} { 
+        r set server_foo server_bar 
+        assert_equal server_bar [run_script_ro {return redis.call('get', KEYS[1]);} 1 server_foo]
+    }
```

Comment:
Test case name doesn't match actual call.

Reply:
Fixed it with new tests